pyogrio/tests/test_raw_io.py

```diff
@@ -270,6 +271,7 @@ def test_write(tmpdir, naturalearth_lowres):
 
 def test_write_gpkg(tmpdir, naturalearth_lowres):
     meta, _, geometry, field_data = read(naturalearth_lowres)
+    meta.update({"geometry_type": "MultiPolygon"})
```

Comment:
The problem here is that reading the Shapefile will indicate the geometry type is "Polygon" (because shapefiles don't care about the distinction), but then using that `meta` to write to geopackage will raise a warning because geopackage is stricter